interface/cerrors/pokemon_errors.go

```diff
@@ -0,0 +1,23 @@
+package cerrors
+
+type RequestError struct {
+	Message  string
+	HttpCode int
```

Comment:
Rename to HTTPCode for better readability.

```suggestion
	HTTPCode int
```